satzung.md

```diff
@@ -70,7 +70,21 @@ Zu den Aufgaben des Fachschaftsrates gehören insbesondere:
 (5) Näheres regelt die Geschäftsordnung des Fachschaftsrates.
 
 
+## § 4c Fachschaftsrats-Assoziierte
+
+(1) Fachschaftsrats-Assoziierte sind Personen, die den Fachschaftsrat in bestimmten Aufgaben unterstützen. Dies ist nicht möglich bei Ämtern und Aufgaben, die nach der Geschäftsordnung des Fachschaftsrates von dessen Mitgliedern übernommen werden müssen.
+
+(2) Fachschaftsrats-Assoziierte werden durch den Fachschaftsrat benannt.
```

Comment:
Ist gegeben, dass diese Bennung protokolliert wird? Wird entweder durch meinen nächsten Vorschlag geheilt, sonst sollten wir das noch explizit festhalten (oder ich übersehe gerade was, dann ist auch ok)

Reply:
Soweit ich weiß, bedarf es bei der Benennung einen Beschluss. Vielleicht weiß @janno42 mehr?

Reply:
§ 2 GO fordert keinen Beschluss, weil dort Wahlen beschrieben sind, die sich in ihrer Art von Beschlüssen unterscheiden. Dass Benennungen einen Beschluss erfordern ist soweit ich sehe bislang nicht explizit gefordert und ließe sich damit höchstens als übliche Vorgehensweise fordern.
Eine gute Lösung wäre bspw., § 10 GO (1) 5. um Wahlen und Benennungen zu ergänzen.

Reply:
@arminWel Siehe Commit zu der Sache. Löst das den Kommentar?